documents/operating.md

```diff
@@ -11,6 +12,17 @@ Contents:
 6.  Policies
 
 ---------------------------------------------------------------------
+SECTION 0: 📓📓📓 Definitions 📓📓📓
+---------------------------------------------------------------------
+
+* **Bad Idea Factory Limited Liability Company ("the Company")** is a Limited Liability Company domiciled in Pennsylvania.
+* The Company may also be referred to as "**Bad Idea Factory**" or "**BIFFUD**".
+* "**Corporate Overlord**" shall mean an "owner" of the Company under Pennsylvania state law.
+* The "**Board of Directors**" ("the **Board**") of the Company, as defined by Pennsylvania state law, shall consist of all current Corporate Overlords of the Company.
+* "**Consensus**" shall mean TKTKTK.
+* "**Quorum**" shall mean TKTK
```

Comment:
so basically @slifty's solution to this is that we define any Consensus (for governance / voting purposes) to always be based on the # of current CO's (not simply participating or present). which sounds good to me.

in terms of a quorum being required to maintain the existence of the organization (i.e. we need "a quorum", or something like it, to meet at some minimum interval), we still need that. OR we rewrite the criteria in "Company Term".

that is the only reason the definition of "quorum" is here. again let us not get caught up in all of the possibilities of the term, for now just consider the one place we use the term.